src/collector.ts

```diff
@@ -105,9 +105,33 @@ class NaiveGomodParser {
     dependencies: Array<IDependency>;
 
     static parseDependencies(contents:string, goImports: Set<string>): Array<IDependency> {
+        let replaceDependencies = new Map();
         let goModDeps = contents.split("\n").reduce((dependencies, line, index) => {
-            // Ignore "replace" lines
-            if (!line.includes("=>")) {
+            if (line.includes("=>")) {
+                // skip any text after '//'
+                if (line.includes("//")) {
+                    line = line.split("//")[0];
+                }
+                // split the replace statements by '=>'
+                const parts: Array<string> = line.replace('replace', '').replace('(', '').replace(')', '').trim().split(' => ');
+                const replaceTo: Array<string> = (parts[0] || '').trim().split(' ');
+                const replaceWith: Array<string> = (parts[1] || '').trim().split(' ');
+                regExp.lastIndex = 0;
+                const replaceWithVersion = regExp.exec(parts[1]);
+                const replaceWithIndex = line.indexOf(parts[1]);
```

Comment:
Use lastIndexOf() to get proper index in case replaceIn and replaceWith are same.